lib/repl.js

```diff
@@ -131,7 +131,7 @@ const { shouldColorize } = require('internal/util/colors');
 const CJSModule = require('internal/modules/cjs/loader').Module;
 let _builtinLibs = ArrayPrototypeFilter(
   CJSModule.builtinModules,
-  (e) => e[0] !== '_',
+  (e) => e[0] !== '_' && !StringPrototypeStartsWith(e, 'node:'),
```

Comment:
I guess exposing these to the REPL would actually be good, while it's also good to have it in a separate PR lands, to clearly separate the things from each other.

Reply:
happy to do that as a followup.